config/babel/.babelrc

```diff
@@ -1,4 +1,22 @@
 {
-  "stage": 0,
-  "nonStandard": true
+  "presets": ["react"],
+  "plugins": [
+    "transform-class-properties",
+    "transform-es2015-arrow-functions",
+    "transform-es2015-block-scoping",
+    "transform-es2015-classes",
+    "transform-es2015-destructuring",
+    "transform-es2015-shorthand-properties",
+    "transform-object-rest-spread"
```

Comment:
What's the reason for using all of these plugins over `babel-preset-env`?

Reply:
I was mostly copying from `builder-victory-component` which originally had `react` (good) and `react-native`, which was hodgepodge of beyond the kitchen sink of stuff we didn't need. So in bvc, I just honed down to the plugins actually needed beyond the `preset`.

So, that's the long way of asking if you know of a preset that would be pretty close to this minimal set of stuff for a reasonable, modern react component? (And ideally without getting into an experimental territory -- we're still unwinding Radium's early bad bet on now deprecated early stage decorators 😛  )

Thanks!

Reply:
I would recommend [`babel-preset-env`](https://github.com/babel/babel-preset-env). It's being positioned as the long-term solution for Babel preset/plugin fatigue 😄 Instead of having to explicitly add/remove plugins the preset will detect which features you need to compile based on your target environment. The other standard presets like `babel-preset-es2015` are being deprecated in favor of `babel-preset-env` AFAIK.

In the context of builder, it could allow an API for per-project environment configuration (with a sane default), so individual projects could update their targets without depending on an archetype update.

Reply:
@aweary -- What configuration would you recommend? Offhand, the default (everything) looks like a **monster** list: https://github.com/babel/babel-preset-env/blob/master/package.json#L28-L62

Reply:
@ryan-roemer with `babel-preset-env` your configuration is just your target environment, e.g., `"last 2 versions, ie 10"`. This is done via [browserlist](https://github.com/ai/browserslist) which is used by other tools like autoprefixer as well. 

By default `babel-preset-env` will be equivalent to `babel-preset-e2015`, `babel-preset-es2016`, and `babel-preset-es2017`, which only contain plugins for features that have been fully approved by TC39 (no stage 0-3 features).

Reply:
Cool. Updated the archetype to `babel-preset-env` and then added the two extra plugins not used there specifically to `radon-typeahead`. Seems like a sensible default for the archetype.

Thanks!